proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,155 @@
+# Instructor Training Selection
+
+**Warning: This is only a draft.**
+
+**Warning: This only applies for the two-day training course in December 2015.**
+
+## Terminology
+
+-   An **instructor** is someone who teaches a workshop.
+    Instructor training (IT) prepares them to do this.
+
+## Why Select Attends for Instructor Training
+
+SCF wants to train as many instructors as possible
+but demand for instructor training is higher than what SCF is able to offer
+at the moment.
+
+## Eligibility
+
+We expect all students to gather in groups
+that are able to co-locate in one room for the duration of the Instructor Training.
+Each group should commit to jointly run one workshop no later than April 1st, 2016.
+**Please note: how to run a workshop is covered in the curriculum.**
+
+### Participation in a group
+
+The members of the group should be on the current waiting list
+or
+demonstrate sufficient competence with core tools used in the lessons they will teach.
+
+Each student is expected to:
+
+-   Watch one of the two videos below
+    and submit a paragraph of feedback on the video they chose to watch
+    as part of their application.
+
+    - https://vimeo.com/139316669
+    - https://vimeo.com/139181120
+
+-   **Only** students interested in teach for Software Carpentry need to
+    submit a pull request to the GitHub repository at
+    https://github.com/swcarpentry/training-prep-2015-09.
+
+    Your pull request should contain one file (and one file only) called
+    "your_name.txt" (e.g., marie_curie.txt or alan_turing.txt). This file
+    should contain a sentence or two that describes the type of work you do.
+    You will submit the name of this file as part of your application.
+
+    Help on creating pull requests can be found at
+    https://help.github.com/articles/using-pull-requests/.
+
+    If you need assistance, please mail training-help@software-carpentry.org.
+
+## Application
+
+Groups **must** fill
```

Comment:
change to `fill out [this survey](https://docs.google.com/forms/d/1Fzm6HQt5RWyzbCSMmwj_8G78vlP7kXlT5sGjARXhjYM/edit?usp=sharing)`